lib/ssl/src/tls_server_session_ticket.erl

```diff
@@ -148,20 +155,23 @@ format_status(_Opt, Status) ->
 %%% Internal functions
 %%%===================================================================
 
-inital_state([stateless, Lifetime, _, MaxEarlyDataSize, undefined]) ->
+inital_state([stateless, Lifetime, _, MaxEarlyDataSize, undefined, Seed]) ->
     #state{nonce = 0,
-           stateless = #{seed => {crypto:strong_rand_bytes(16), 
-                                  crypto:strong_rand_bytes(32)},
+           stateless = #{seed => stateless_seed(Seed),
                          window => undefined},
            lifetime = Lifetime,
            max_early_data_size = MaxEarlyDataSize
           };
-inital_state([stateless, Lifetime, _, MaxEarlyDataSize, {Window, K, M}]) ->
+inital_state([stateless, Lifetime, _, MaxEarlyDataSize, {Window, K, M}, Seed]) ->
     erlang:send_after(Window * 1000, self(), rotate_bloom_filters),
     #state{nonce = 0,
            stateless = #{bloom_filter => tls_bloom_filter:new(K, M),
-                         seed => {crypto:strong_rand_bytes(16),
-                                  crypto:strong_rand_bytes(32)},
+                         %% Warm up for 2*WindowSize to ensure tickets from a
+                         %% previous instance of the server (before a restart)
+                         %% cannot be reused, if the ticket encryption seed is
+                         %% reused
+                         warm_up_windows_remaining => 2,
```

Comment:
can you make initial value 0 for legacy runs (when new option is not specified)?
and 2 for flows with hardcoded random - i.e. new option is specified?

we discussed it, and think this is preferred. not only it solves test case issues but also make PR not changing legacy flows.

Reply:
Sure, makes sense. Done.